src/response/generate_api_token_response.rs

```diff
@@ -0,0 +1,17 @@
+use std::time::SystemTime;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ApiKeyWithEndpoint {
+    pub api_key: String,
+    pub endpoint: String,
+}
+
+/// The result of a generate api token operation.
+#[derive(Debug)]
+pub struct MomentoGenerateApiTokenResult {
```

Comment:
In Rust, `Result` evokes thoughts of the `Result` type. Often times people will write their own Error type and define a *Result to wrap it up.

By my read in 9 out of 10 of the `_response` files in this `response` module, the response type is called a `Response`. Is there some reason I'm missing to keep this named "result?"

Reply:
good call

Reply:
fixed in https://github.com/momentohq/client-sdk-rust/pull/132/commits/a10baf2df4d874788a9eae80e49a54fbedf939bd